### ISSUE TITLE
doc: example for manifest from file is wrong

### ISSUE THREAD

**Issue by tpolecat** [sbt/sbt#1072]
The example at http://www.scala-sbt.org/0.13.1/docs/Howto/package.html#manifest for "read the manifest from a file" doesn't compile. This

``` scala
Using.fileInputStream( in => new java.util.jar.Manifest(in) )
```

should be something like 

``` scala
Using.fileInputStream(baseDirectory.value / "stuff.mf") { 
  in => new java.util.jar.Manifest(in)
}
```

**Comment by jsuereth** [comment]
Thanks for the report.  Feel free to submit a patch if you'd like!

**Comment by dwijnand** [comment]
Fixed in sbt/website#192.
